app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import shlex
+import os
+
+
+def move_file(command: str) -> None:
+    if command[:3] != "mv ":
```

Comment:
why not simplify the whole check process? 
You can split the file first (by the way why do you choose to use shlex.split() instead of built-in split()?)
Then you check if "mv" is there if len(command) is ok and put your main logic below

Reply:
@Oleksl888 
> why not simplify the whole check process? You can split the file first (by the way why do you choose to use shlex.split() instead of built-in split()?) Then you check if "mv" is there if len(command) is ok and put your main logic below

I chose shlex.split() because of the task of writing a similar command to the "mv" in bash, if there are spaces in the file, they can be escaped with \ or the full file name with spaces will be enclosed in quotes, for built-in split() you will need additional logic, and in shlex.split() it was already written